packages/block-library/src/heading/heading-toolbar.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { range } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { Toolbar } from '@wordpress/components';
+
+class HeadingToolbar extends Component {
+	createLevelControl( targetLevel, selectedLevel, onChange ) {
```

Comment:
Minor improvement, but you can read `{ selectedLevel, onChange }` directly from `this.props` instead of passing it as arguments.